### PR TITLE
Generate YAML / python files for gas_phase_chemdr, setinv; remove unused setinv code

### PR DESCRIPTION
**Pull request by brigaudet**
1)  Generate YAML / python files for gas_phase_chemdr, setinv subroutines.

2)  Remove code in setinv unused in our tests, and rename local index variables.

Note:  pbuf is an input to gas_phase_chemdr, but is only used after passed to child subroutine setinv. Thus, its relevant input fields are only written out in the setinv files, by extracting them in a manner similar to the actual code.

### CODE REVIEW

File: components/eam/src/chemistry/yaml/mo_setinv/setinv_output_ts_1415.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+input.boltz_cgs=[[  0.13806500000000001E-015],]
+input.num_tracer_cnst=[[         4],]
+input.has_n2=[[         1],]
+input.m_ndx=[[         1],]
+input.n2_ndx=[[         2],]
+input.has_o2=[[         1],]
+input.o2_ndx=[[         3],]
+input.has_h2o=[[         1],]
+input.h2o_ndx=[[         4],]
+input.tracer_cnst_flds=[[cnst_O3,OH,NO3,HO2,],]
```
Comment (jeff-cohere) [inline components/eam/src/chemistry/yaml/mo_setinv/setinv_output_ts_1415.py]: These names need to be quoted so they're Python strings, like

```
input.tracer_cnst_flds=[['cnst_O3','OH','NO3','HO2',],]
```

Comment (jeff-cohere) [reply components/eam/src/chemistry/yaml/mo_setinv/setinv_output_ts_1415.py]: Unfortunately, though, Skywalker won't be able to use string inputs, so we'll likely have to hardwire these species into our related tests.

Comment (brigaudet) [reply components/eam/src/chemistry/yaml/mo_setinv/setinv_output_ts_1415.py]: OK, thanks.  Then should I remove them from the *.py files?  (and the *.yaml files as well?)

Comment (jeff-cohere) [reply components/eam/src/chemistry/yaml/mo_setinv/setinv_output_ts_1415.py]: Sure, that's probably best. I'll make a note of the species list.

Comment (brigaudet) [reply components/eam/src/chemistry/yaml/mo_setinv/setinv_output_ts_1415.py]: As long as strings with quotes in them do no harm, I've gone ahead and inserted quotes for the string fields in a new batch of files.  If they do harm things though then I will go ahead and just remove the field.

**Review by jeff-cohere — APPROVED** [review]

**Comment by brigaudet** [comment]
image

All longer tests passed!

**Pull request merged**